core/code/data_cache.js

```diff
@@ -24,11 +24,7 @@ window.DataCache = function() {
 
 }
 
-window.DataCache.prototype.store = function(qk,data,freshTime) {
-  // fixme? common behaviour for objects is that properties are kept in the order they're added
-  // this is handy, as it allows easy retrieval of the oldest entries for expiring
-  // however, this is not guaranteed by the standards, but all our supported browsers work this way
-
```

Comment:
I found some info for es2020 about this, but not strong result for es2015, do you have one ?

Reply:
to tell the truth: no.
I also only have my personal experience and other blog-posts talking about "since ES5 ...". 
Like: https://dev.to/frehner/the-order-of-js-object-keys-458d
Which leads to current spec: https://tc39.es/ecma262/#sec-ordinaryownpropertykeys

On the other hand if the order is random it wouldn't break the cache.
It would just decrease it's performance. If it hits the entry or memory limit (more then 300Tiles or 5MB in 5minuts) it could 'forget' freshly loaded tile.

Reply:
BTW: also overlay creation on map init assumes that object are sorted